decidim-proposals/app/cells/decidim/proposals/proposal_vote/show.erb

```diff
@@ -24,22 +24,23 @@
     <div id="proposal-<%= resource.id %>-vote-button" class="card__proposals-votes-container">
       <% if !current_user %>
         <% if current_settings.votes_blocked? %>
-          <%= action_authorized_button_to :vote, t("decidim.proposals.proposals.vote_button.votes_blocked"), proposal_proposal_vote_path(proposal_id: resource, from_proposals_list:), resource: , class: button_classes, disabled: true %>
+          <%= action_authorized_button_to :vote, t("decidim.proposals.proposals.vote_button.votes_blocked"), proposal_proposal_vote_path(proposal_id: resource, from_proposals_list:), resource: , class: button_classes, disabled: true, data: { "proposal-vote-button": true, disable: true } %>
```

Comment:
I think this `data-disabled="true"` isn't doing anything. Mind that we already have the `disabled: true` just before, and that adds the HTML 5 attribute correctly (`disabled="disabled"`)

```suggestion
          <%= action_authorized_button_to :vote, t("decidim.proposals.proposals.vote_button.votes_blocked"), proposal_proposal_vote_path(proposal_id: resource, from_proposals_list:), resource: , class: button_classes, disabled: true, data: { "proposal-vote-button": true } %>
```